IaC/AKS/README.md

```diff
@@ -391,12 +386,53 @@ Run the Validation Test
 
 > For more information on the validation test tool, see [Web Validate](https://github.com/retaildevcrews/webvalidate)
 
+```bash
+
+# run the tests in a container
+docker run -it --rm retaildevcrew/webvalidate --server $Ngsa_App_Endpoint --base-url https://raw.githubusercontent.com/retaildevcrews/ngsa/main/TestFiles/ --files baseline.json
+
+```
 
 ## Observability
 
 TODO
 
+## Smoke Tests
 
-## Smoke Test
+Deploy Web Validate to drive consistent traffic to the App Service for monitoring and alerting.
 
-TODO
+```bash
+
+# Add Log Analytics extension
+az extension add -n log-analytics
+
+# create Log Analytics for the webv clients
+az monitor log-analytics workspace create -g $Ngsa_Smoker_RG -l $Ngsa_Location -n $Ngsa_Name -o table
+
+# retrieve the Log Analytics values using eval $Ngsa_LogAnalytics_*
+export Ngsa_LogAnalytics_Id='az monitor log-analytics workspace show -g $Ngsa_Smoker_RG -n $Ngsa_Name --query customerId -o tsv'
+export Ngsa_LogAnalytics_Key='az monitor log-analytics workspace get-shared-keys -g $Ngsa_Smoker_RG -n $Ngsa_Name --query primarySharedKey -o tsv'
+
+# create Azure Container Instance running webv
+az container create -g $Ngsa_Smoker_RG --image retaildevcrew/webvalidate:latest -o tsv --query name \
+-n ${Ngsa_Name}-webv-${Ngsa_Location} -l $Ngsa_Location \
+--log-analytics-workspace $(eval $Ngsa_LogAnalytics_Id) --log-analytics-workspace-key $(eval $Ngsa_LogAnalytics_Key) \
+--command-line "dotnet ../webvalidate.dll --tag $Ngsa_Location -l 1000 -s $Ngsa_App_Endpoint -u https://raw.githubusercontent.com/retaildevcrews/ngsa/main/TestFiles/ -f benchmark.json -r --json-log"
```

Comment:
Wasn't sure to move the rest of the steps into a script just like:
https://github.com/retaildevcrews/helium-iac/blob/main/scripts/smokers.sh

Reply:
I think that would make it more usable and easier to read.